File: org.opencompare/api-java/src/main/java/org/opencompare/api/java/io/ExportCell.java
package org.opencompare.api.java.io;

/**
 * Created by gbecan on 08/12/15.
 */
public class ExportCell extends IOCell {

    protected boolean feature;
    protected boolean inProductsKeyColumn;

    public ExportCell() {

    }

    public ExportCell(String content) {
        super(content);
    }

    public ExportCell(String content, String rawContent) {
        super(content, rawContent);
    }

    public ExportCell(String content, int rowspan, int colspan) {
        super(content, rowspan, colspan);
    }

    public ExportCell(String content, String rawContent, int rowspan, int colspan) {
        super(content, rawContent, rowspan, colspan);
    }

    public boolean isFeature() {
        return feature;
    }

    public void setFeature(boolean feature) {
        this.feature = feature;
    }

    public boolean isInProductsKeyColumn() {
        return inProductsKeyColumn;
    }

    public void setInProductsKeyColumn(boolean inProductsKeyColumn) {
        this.inProductsKeyColumn = inProductsKeyColumn;
    }
}

File: org.opencompare/api-scala/src/main/scala/org/opencompare/api/scala/io/ExportCell.scala
package org.opencompare.api.scala.io

class ExportCell(
                  initContent : String = "",
                  initRawContent : String = "",
                  initRowspan : Int = 1,
                  initColspan : Int = 1)
  extends IOCell(initContent, initRawContent, initRowspan, initColspan) {

  var feature : Boolean = false
  var inProductsKeyColumn : Boolean = false
}
